qulice-checkstyle/src/test/resources/com/qulice/checkstyle/MainWindows.java

```diff
@@ -0,0 +1,13 @@
+/**
+ * Hello.
+ *
+ * World.
+ */
+package foo;
+/**
+ * Simple class.
+ * @version $Id $
+ * @author John Doe (john@qulice.com)
+ */
+public class MainWindows { }
```

Comment:
@prahladyeri please rename the class to a more appropriate name, something that will tell me what is tested here